### PR TITLE
added more items

### PR DESCRIPTION
**Pull request by gauravsquadx**

### CODE REVIEW

**Comment by gauravsquadx** [comment]
Comments added

File: JavaScript_Basics/bugs.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+var obj = {
+    name: 'Some name'
+}
+
+var name = null;
+
+console.log(typeof obj) // object;
+console.log(typeof name) // object;
```
Comment (Swap76) [inline JavaScript_Basics/bugs.js]: @gauravsquadx Explanation about this bug will be highly appreciated

**Comment by Swap76** [comment]
@gauravsquadx Merging the PR now if want to add a description of all the examples feel free to add it :-)

**Comment by Swap76** [comment]
@gauravsquadx The work you have done for the project is highly appreciated. Thanks :-)

Looking forward to working with you on my new project related to Deno https://github.com/Swap76/Learn-Deno/ and together making this better.

Star to the repo is highly appreciated.

**Pull request merged**